docs/iciclecharts-with-cpu-profiles.mdx

```diff
@@ -0,0 +1,31 @@
+# Iciclecharts with CPU profiling data
+
+Iciclechart is a visualization method that allows you to see the CPU usage of your application over time. In other words, it is similar to a flamegraph, but has an additonal time dimension. This means that you can see how the CPU usage of your application changes over time, and how it is affected by different events.
+
+Screenshot of Iciclechart with CPU profiling data:
+![Iciclechart with CPU profiling data](/img/vizualisation/iciclechart-example.png)
+
+## Prerequisite
+Before you can use Iciclechart with CPU profiling data, you need to be collecting the CPU profiling data with `cpu` or `thread_id` labels. You can do with by adding the following section to the parca agent configuration if you don't have it already:
```

Comment:
I think we can qualify the labels with ", or any other label that will never have samples with overlapping timestamps, threads or CPUs can't have two things running on them at the same time so they are well suited for this purpose".

Reply:
Done